code/modules/mob/living/carbon/human/ai/brain/ai_brain.dm

```diff
@@ -93,162 +87,135 @@ GLOBAL_LIST_EMPTY(human_ai_brains)
 
 /datum/human_ai_brain/process(delta_time)
 	if(tied_human.is_mob_incapacitated())
-		end_gun_fire()
-		current_target = null
+		for(var/action in ongoing_actions)
+			qdel(action)
+		ongoing_actions.Cut()
+		lose_target()
 		return
 
-	var/order_return
-	if(can_process_order())
-		order_return = ongoing_order.trigger_action(src)
+	lose_target()
```

Comment:
why do we lose our target every process()?

Reply:
Looks better inside the if() block, agreed